src/Analyzers/MSTest.Internal.Analyzers/MSTest.Internal.Analyzers.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <EnforceExtendedAnalyzerRules>true</EnforceExtendedAnalyzerRules>
+    <LangVersion>Latest</LangVersion>
```

Comment:
We are using preview in Directory.Build.props. I think we shouldn't overwrite that?